public_dropin_environments/triton_server/Dockerfile

```
@@ -0,0 +1,24 @@
+# This is the default base image for use with user models and workflows.
+FROM nvcr.io/nvidia/tritonserver:24.02-py3
+
+# Install the list of core requirements, e.g. sklearn, numpy, pandas, flask.
+# **Don't modify this file!**
+COPY dr_requirements.txt dr_requirements.txt
+
+# '--upgrade-strategy eager' will upgrade installed dependencies
+# according to package requirements or to the latest
+RUN pip3 install -U pip && \
+    pip3 install -U --upgrade-strategy eager --no-cache-dir -r dr_requirements.txt  && \
```

```suggestion
    pip3 install -U --upgrade-strategy eager --no-cache-dir -r dr_requirements.txt  && \
```
`-U --upgrade-strategy eager` is a little dangerous right? Since we don't control the upstream env I think we should be as isloated as possible.

---

I don't think we need the "eager" strategy here. It's here because I just reused the Dockerfile from existing dropin env. Will drop